engine/standalone/app/src/test/scala/pl/touk/nussknacker/engine/standalone/http/TestConfigCreator.scala

```diff
@@ -19,7 +19,8 @@ class TestConfigCreator extends EmptyProcessConfigCreator {
   override def sourceFactories(processObjectDependencies: ProcessObjectDependencies): Map[String, WithCategories[SourceFactory[_]]] = Map(
     "request1-post-source" -> WithCategories(new JsonStandaloneSourceFactory[Request]),
     "request1-get-source" -> WithCategories(RequestGetSourceFactory),
-    "genericGetSource" -> WithCategories(new TypedMapStandaloneSourceFactory)
+    "genericGetSource" -> WithCategories(new TypedMapStandaloneSourceFactory),
```

Comment:
Can we remove/merge it with JsonSchemaStandaloneSourceFactory? It's more or less the same, but for GET

Reply:
I do not see point in merging it with `jsonSchemaSource` tests since it not provides any extra value. I do not want to remove it either since it is the only test for generic GET requests. It is not complex so I suggest to keep it.